### DISCUSSION TITLE
Absolute positioning which wiggles and jiggles.

### DISCUSSION THREAD

**Discussion by braindeaf** [orchidjs/tom-select#372] [Ideas]
We've recently looked at moving away from Chosen / Select2 to TomSelect and in terms of UI it's so much nicer than the previous two plus we can remove a heavy dependency on jQuery which will benefit us in time. However, we noticed when we tried to use it in a table that expanding the select meant that the table cell grew rather than overlaying it. I can see one way of getting round this is to mark the dropdownParent as 'body' and then absolute positioning places the faux select nicely where the original select was with the correct width etc. 

However...if that select is within a scrollable area on the page then this breaks because the absolute positioning is relative to window and that position will change if something in the page scrolls and then the select is left hanging in a weird place. I've seen that select2 copes with this fine and indeed it has the further ability to flip to open upwards if the select would drop out of sight rather like a native select. Are there any plans to implement this? any advice on where to hook in to get this positioning to change on any kind of scroll event?

Many thanks

RobL

**Comment by oyejorge** [comment]
Probably need to add this to the docs. The solution for opening upwards (and possibly the wiggle/jiggles) is to use popper. [For example](https://jsfiddle.net/ok56a0gu/)

**Reply by braindeaf** [reply]
Thank you. I took a look, it still behaves badly in tables. And sadly there is a weird scroll issue when it does open. It might all be our side though so I'll take a closer look in the morning.

Thank you for getting back to me :)
